### PR TITLE
community: add blog, improve welcome

### PR DESCRIPTION
**Pull request by debs-obrien**
- remove github issues and add info of help on discord server
- add blog link to footer and sidebar
- improve wording on welcome page
- add discord logo to header

### CODE REVIEW

File: dotnet/docusaurus.config.js [hunk 1]
```diff
@@ -79,11 +79,17 @@ module.exports = {
           position: "left",
         },
         {
-          href: "https://github.com/microsoft/playwright-dotnet",
+          href: "https://github.com/microsoft/playwright",
```
Comment (mxschmitt) [inline dotnet/docusaurus.config.js:82]: why remove dotnet here?

Comment (debs-obrien) [reply dotnet/docusaurus.config.js:82]: ooops

**Review by mxschmitt — APPROVED** [review]
lgtm except removing the links to the language binding repos.

File: dotnet/docusaurus.config.js (hunk 1 above)
Comment (mxschmitt) [reply dotnet/docusaurus.config.js:82]: the same applies to java and python

File: community/0-welcome.mdx [hunk 2]
```diff
@@ -14,27 +14,28 @@ Please create an issue for the following:
...
-Or why not join our [GitHub Discussions community](https://github.com/microsoft/playwright/discussions)
-
 ## Contributing
 
 Check out our [contributing guide](https://github.com/microsoft/playwright/blob/main/CONTRIBUTING.md) if you would like to contribute to Playwright.
 
 ## Community Discord
 
-Join our community [Discord Server](https://aka.ms/playwright/discord) to connect with other developers using Playwright.
+Join our community [Discord Server](https://aka.ms/playwright/discord) to connect with other developers using Playwright, ask questions in our 'help-playwright' forum, share your articles and videos and join live events on the Playwright stage.
```
Comment (mxschmitt) [inline community/0-welcome.mdx:26]: we could link here to the help-playwright forum, I think thats a direct link: https://discord.com/channels/807756831384403968/1054804523652231198

**Review by mxschmitt — APPROVED** [review]

**Pull request merged**